app/models/room.rb

```diff
@@ -0,0 +1,47 @@
+
+class Room < ActiveRecord::Base
+  belongs_to :week
+  belongs_to :user
+  has_many :rooms_learning_objects
+  has_many :user_to_lo_relation
+  has_many :learning_objects, through: :rooms_learning_objects
+
+  # Vrati learning objecty, ktore v danom pokuse a miestnosti este neboli videne
+  def get_dont_visited
+    visited_los = self.learning_objects.eager_load(:user_to_lo_relations).where("user_to_lo_relations.room_id = ? AND user_to_lo_relations.number_of_try = ?",self.id,self.number_of_try)
+    los = self.learning_objects
+    los_result = los.reject{ |l| visited_los.include?(l) }
+    los_result
+  end
+
+  def question_count
+    self.learning_objects.count
+  end
+
+  # Vrati pocet vyriesenych otazok v danej miestnosti
+  def question_count_done user_id
+    lo_ids = self.learning_objects.map(&:id)
+    UserSolvedLoRelation.where(learning_object_id: lo_ids, user_id: user_id).
+        group(:learning_object_id).count.
+        count
```

Comment:
Mohla by to byt ok, lebo question_count je pocet vsetkych otazok v miestnosti a question_count_done je pocet uz vyriesenych otazok v miestnosti. 
Pouzivam to pri progresbare:  <div class="room-progressbar" data-number-all="<%= r.question_count %>" data-number-done="<%= r.question_count_done current_user.id %>">